composer.json

```diff
@@ -19,7 +19,8 @@
     },
     "require": {
         "php": ">=5.4",
-        "beberlei/assert": "^2.6"
+        "beberlei/assert": "^2.6",
+        "tomphp/exception-constructor-tools": "^0.1.0"
```

Comment:
@tomphp 

I trust that you trust the maintainer, hehe, but maybe we should pin to a concrete version, since we have no stable release yet?